tests/unit/Hooks/ImageHooksTest.php

```diff
@@ -92,7 +95,9 @@ public function testPostCreate() {
 		$imageHeight = 100;
 		$dimensions = array($imageWidth, $imageHeight);
 
-		$this->mockGetImageSize($this->imagehooks, $location, $dimensions);
+		$this->mockImageHooks = $this->mockPostCreate(['postCreate',]);
```

Comment:
Okay, I will remove that but then how can I check whether getimagesize is called exactly once or not? I am not able to figure out that.